curl.yaml

```diff
@@ -22,26 +21,17 @@ environment:
       - openssl-dev
       - zlib-dev
       - brotli-dev
-
 pipeline:
   - uses: fetch
     with:
       uri: https://curl.se/download/curl-${{package.version}}.tar.xz
       expected-sha256: 88b54a6d4b9a48cb4d873c7056dcba997ddd5b7be5a2d537a4acb55c20b04be6
   - uses: autoconf/configure
     with:
-      opts: |
-        --enable-ipv6 \
-        --enable-unix-sockets \
-        --with-openssl \
-        --with-nghttp2 \
-        --with-pic \
-        --disable-ldap \
-        --without-libssh2    
+      opts: "--enable-ipv6 \\\n--enable-unix-sockets \\\n--with-openssl \\\n--with-nghttp2 \\\n--with-pic \\\n--disable-ldap \\\n--without-libssh2    \n"
```

Comment:
This is sort of unfortunate 😒

Reply:
Can we not? :)

Reply:
Yeah, let's not.

The easiest way will be to just have it be a single line, which is unambiguously formatted. Let me know if you'd like me to figure out how to get the newlines preserved instead.

Reply:
I would prefer to keep the newlines as they are.

Reply:
I would also prefer that. Unfortunately I can't figure out how to get `yq` to preserve those. Short of writing our own yamlfmt or adding support for this to yq, I think it's a worthy sacrifice if it means we get linting and bug catching in place.